main.go

```diff
@@ -7,6 +7,11 @@ func Divine()  {
 func Add2()  {
 	
 }
+
+func Add3()  {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`Add3` is unused (deadcode)